gamestonk_terminal/stocks/fundamental_analysis/dcf_view.py

```diff
@@ -77,8 +77,7 @@ def create_workbook(self):
             self.run_audit()
 
         trypath = os.path.join(
-            "..",
-            "GamestonkTerminal",
+            os.getcwd(),
```

Comment:
I would recommend something like this instead: 
```python
os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
```

This is because with `os.getcwd()` you rely on the location of where you execute terminal. Which in 99% of the cases will be `python terminal.py` because you are on the same directory of `terminal.py`. However, let's say you are in an upper folder, you can run the terminal with `python GamestonkTerminal/terminal.py` in those cases `os.getcwd()` would retrieve directory above which will make this path to not exist.

Thus, to be 100% correct we can rely on the location of this file as the "anchor" and therefore regardless of what the user does, it should work

Reply:
Added this fix. Unfortunately it required a bit more than just this line. This is because with 
```
os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
```

You select the stocks directory while you want the main directory. This results in this line:
```
  os.path.abspath(
      os.path.join(
          os.path.dirname(os.path.dirname(os.path.dirname(__file__))), ".."
      )
  )
```

Reply:
I said something like this because I used this Sunday on the discordbot :)

Something that would work in your case and is more elegant would be:
```
os.path.abspath(
   os.path.join(
      os.path.dirname(__file__),
      "..",
      "..", 
      ".."
   )
)
```

This is how it works:
*  os.path.dirname(__file__): directory of where the current file is (__file__), which is dcf_view.py and therefore fundamental_analysis.
* When you do os.path.join, you are just adding "/" or "\" depending on OS, therefore:
   * with 1 `..` you get into stocks (1 level above) 
   * with 2 `..` you get into gamestonk_terminal (2 levels above) 
   * with 3 `..` you get into GamestonkTerminal (3 levels above) 
* The absolute path just simplifies it to it's simplest form removing those `..` and effectively getting the absolute path